pypeit/calibrations.py

```diff
@@ -190,11 +186,9 @@ def set_config(self, frame, det, par=None):
         the internals to None. The internal dict is left unmodified.
 
         Args:
-            frame (int):
-            det (int):
-            par (CalibrationPar):
-
-        Returns:
+            frame (int): Frame index in the fitstbl
+            det (int): Detector number
+            par (pypeit.par.pypeitpar.CalibrationPar):
```

Comment:
Missing `:obj:` or `:class:`.

Reply:
PS - That's how sphinx knows to link the object here to its definition elsewhere in the documentation.

Reply:
got it.  Note I don't think `obj` is needed for defaut types like `str`, `int`.